torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2089,6 +2112,28 @@ def _make_tensor_helper(shape, low=None, high=None):
                         # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
                         SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),)
                     ),
+    OpInfo('linalg.det',
+           op=torch.linalg.det,
+           dtypes=floating_and_complex_types(),
+           aten_name='linalg_det',
+           sample_inputs_func=sample_inputs_linalg_det,
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
```

Comment:
Maybe add here `skipCUDAIfRocm` instead of ROCm skips below.

Reply:
we don't want to skip all ROCm tests, only the few which actually fail